README.md

```diff
@@ -121,6 +121,15 @@ Ensure whether the service is running or stopped. If you're passing `absent` to
 |-----|--------|
 | Enum['running', 'stopped'] | `running` |
 
+#### service_enable
+
+Set whether the service is enabled or not. If you're passing `absent` to
+`ensure`, please also pass `false` to `service_enable`.
```

Comment:
we should enforce this either by throwing an error or always setting `$service_enable = false if ensure => absent`